app/config/variables.php

```diff
@@ -818,13 +818,58 @@
             ],
             [
                 'name' => 'OPEN_RUNTIMES_NETWORK',
-                'description' => 'The docker network used for communication between the executor and runtimes. Change this if you have altered the default network names.',
+                'description' => 'Deprecated with 1.2.0, use \'OPR_EXECUTOR_NETWORK\' instead!',
                 'introduction' => '0.13.0',
                 'default' => 'appwrite_runtimes',
                 'required' => false,
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => 'OPR_EXECUTOR_INACTIVE_TRESHOLD',
+                'description' => 'The minimum time a function can be inactive before it\'s container is shutdown and put to sleep. The default value is 60 seconds',
+                'introduction' => '1.2.0',
+                'default' => '60',
+                'required' => false,
+                'question' => '',
+                'filter' => ''
+            ],
+            [
+                'name' => 'OPR_EXECUTOR_NETWORK',
+                'description' => 'The docker network used for communication between the executor and runtimes. Change this if you have altered the default network names.',
+                'introduction' => '1.2.0',
+                'default' => 'appwrite_runtimes',
+                'required' => false,
+                'question' => '',
+                'filter' => ''
+            ],
+            [
+                'name' => 'OPR_EXECUTOR_DOCKER_HUB_USERNAME',
```

Comment:
I think we should keep these variables with the _APP prefix and only in the docker compose replace them with the actual

`OPR_EXECUTOR_DOCKER_HUB_USERNAME=$_APP_DOCKER_HUB_USERNAME`

Reply:
Same with all the new variables

Reply:
Done.